cli/lib/kontena/cli/nodes/ssh_command.rb

```diff
@@ -15,7 +15,14 @@ class SshCommand < Kontena::Command
     requires_current_grid
 
     def execute
-      node = client.get("nodes/#{current_grid}/#{node_id}")
+      if node_id
+        node = client.get("grids/#{current_grid}/nodes/#{node_id}")
+      elsif any?
+        nodes = client.get("grids/#{current_grid}/nodes")['nodes']
+        node = nodes.select{ |node| node['connected'] }.first
+      else
+        exit_with_error "No host node given, nor --any"
```

Comment:
perhaps something like `Missing node name. Use --any to connect to the first available node`